### PR TITLE
feat(example): add video type & example

### PR DESCRIPTION
**Pull request by goldenxinxing**
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [x] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: example/ucf101/ucf101/dataset.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import typing as t
+from pathlib import Path
+
+from starwhale import Video, MIMEType, BuildExecutor
+
+dataset_dir = Path(__file__).parent.parent / "data" / "UCF-101"
```
Comment (tianweidut) [inline example/ucf101/ucf101/dataset.py:6]: add a `root_dir` var?

```python
root_dir = Path(__file__).parent.parent
dataset_dir = root_dir / "data" / "UCF-101"
```

File: example/ucf101/ucf101/evaluator.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        for name, param in state_dict.items():
+            if name in model.state_dict().keys():
+                dst_param_shape = model.state_dict()[name].shape
+                if param.shape == dst_param_shape:
+                    model.state_dict()[name].copy_(param.view(dst_param_shape))
+                    net_state_keys.remove(name)
+        # indicating missed keys
+        if net_state_keys:
+            logging.warning(f">> Failed to load: {net_state_keys}")
+            return False
```
Comment (tianweidut) [inline example/ucf101/ucf101/evaluator.py:34]: replace `return False` with `raise Exception` ?

File: example/ucf101/ucf101/smpler.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+from typing import Any, List, Union
```
Comment (tianweidut) [inline example/ucf101/ucf101/smpler.py:1]: `smpler.py` -> `sampler.py` ?

Comment (goldenxinxing) [reply example/ucf101/ucf101/smpler.py]: fixed

File: example/ucf101/ucf101/evaluator.py (hunk 2 above)
Comment (goldenxinxing) [reply example/ucf101/ucf101/evaluator.py]: fixed

File: example/ucf101/ucf101/dataset.py (hunk 1 above)
Comment (goldenxinxing) [reply example/ucf101/ucf101/dataset.py]: done

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**